build/pkgs/ipython/distros/conda.txt

```diff
@@ -1 +1,2 @@
-ipython
+ipython>=7.13.0,<8.9.0
+# ipython >= 8.9.0 requires prompt_toolkit too new for Sage
```

Comment:
Shouldn't that be a constraint on prompt_toolkit?

Reply:
That's now #36659